config/config.go

```diff
@@ -0,0 +1,52 @@
+package config
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+
+	"github.com/davecgh/go-spew/spew"
+
+	"github.com/spf13/viper"
+)
+
+type config struct {
+	Database struct {
+		User                 string
+		Password             string
+		Net                  string
+		Addr                 string
+		DBName               string
+		AllowNativePasswords bool
+		Params               struct {
+			ParseTime string
+		}
+	}
+	Server struct {
+		Address string
+	}
+}
+
+var C config
```

Comment:
By convention, all the descriptions should start with the name of the struct, interface, function, etc name
Example:
`// C - Description`